### PR TITLE
removed curator from system_tests

### PR DESCRIPTION
**Pull request by Hitenjain14**
### Changes
Removed curator functionality . 

### Related PR's - 
- 0chain - https://github.com/0chain/0chain/pull/2184
- gosdk - https://github.com/0chain/gosdk/pull/819
- zboxcli - https://github.com/0chain/zboxcli/pull/372

### CODE REVIEW

File: tests/cli_tests/zboxcli_add_remove_curator_test.go [hunk 1]
```diff
@@ -1,392 +1 @@
 package cli_tests
```
Comment (Kishan-Dhakan) [inline tests/cli_tests/zboxcli_add_remove_curator_test.go:1]: This file can be deleted, please remove it completely

**Review by Kishan-Dhakan — CHANGES_REQUESTED** [review]

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | rm_curator  |
| gosdk  | NOT OVERRIDDEN  |
| zboxcli  | rm_curator  |
| zwalletcli  | staging  |
| 0chain  | rm-curator  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | master  |
| 0proxy  | staging  |
| 0search  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [success] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | rm_curator  |
| gosdk  | NOT OVERRIDDEN  |
| zboxcli  | rm_curator  |
| zwalletcli  | staging  |
| 0chain  | rm-curator  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | master  |
| 0proxy  | staging  |
| 0search  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | ABILITY TO SKIP TESTS IS DISABLED. SKIP_TESTS OPTION IGNORED.  |
| system-tests  | rm_curator  |
| gosdk  | rm_curator  |
| zboxcli  | rm_curator  |
| zwalletcli  | staging  |
| 0chain  | staging  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | staging  |
| 0proxy  | staging  |
| 0search  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [success] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | ABILITY TO SKIP TESTS IS DISABLED. SKIP_TESTS OPTION IGNORED.  |
| system-tests  | rm_curator  |
| gosdk  | rm_curator  |
| zboxcli  | rm_curator  |
| zwalletcli  | staging  |
| 0chain  | staging  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | staging  |
| 0proxy  | staging  |
| 0search  | staging  |

**Pull request merged**